spec/features/user_can_create_glossary_entries_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'sign_up_helper'
```

Comment:
If you configure `spec/support/**.*.rb` as a pattern, these helpers should be picked up automatically and you don't need to `require` them.